src/steps/when.js

```diff
@@ -104,4 +104,9 @@ defineSupportCode(({ When }) => {
         /^I move to element "([^"]*)?"(?: with an offset of (\d+),(\d+))*$/,
         moveToElement
     );
+
+    When(
+       /^I use the browserfunctionality to (refresh the page|go back|go forward|close the browser)$/,
```

Comment:
I am not sure if this makes sense. I rather would like to have steps like:
- When I refresh the page
- When I go back
- When I go forward
- etc.